src/coreComponents/mpiCommunications/MpiWrapper.hpp

```diff
@@ -389,6 +389,21 @@ struct MpiWrapper
   template< typename T >
   static void broadcast( T & value, int srcRank = 0, MPI_Comm comm = MPI_COMM_GEOSX );
 
+  /**
+   * @brief Strongly typed wrapper around MPI_Scatter()
+   * @tparam T the type to scatter
+   * @param[in] sendValues the values to scatter across ranks
+   * @param[out] myValues the arrayView in which I receive my values
+   * @param[in] srcRank The rank of the source process within \p comm.
+   * @param[in] comm The handle to the MPI_Comm
+   * @return
+   */
+  template< typename T >
+  static int scatter( arrayView1d< T const > sendValues,
+                      arrayView1d< T > myValues,
```

Comment:
Here and below. Probably `destValues` or `recvValues`?

Reply:
Done, I renamed `myValues` to `recvValues`, and in the serial implementation I now copy `sendValues` into `recvValues`.